web/src/features/panels/LeftPanel.tsx

```diff
@@ -97,9 +97,10 @@ function MobileHeader() {
 
 function OuterPanel({ children }: { children: React.ReactNode }) {
   const [isOpen, setOpen] = useAtom(leftPanelOpenAtom);
-  const onCollapse = () => setOpen(!isOpen);
   const location = useLocation();
 
+  const onCollapse = () => setOpen(!isOpen);
+
```

Comment:
Im curious, doe the location of this matter? or was it just a preference change?

Reply:
Doesn't matter 😆 I originally thought the fix might come from this file so I had changed it a bit. My ordering logic was bunch together variable definitions then have a line between functions defined within a component